CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/thirdparty" ${CMAKE_MODULE_PA
 
 # Build options
 option(RAJA_ENABLE_OPENMP "Build OpenMP support" On)
+option(RAJA_ENABLE_TBB "Build TBB support" On)
```

Comment:
Ok, for now I'm going to leave it, but if anyone runs into any issues with it making this off by default before it goes to master would be fine.